templates/index.html

```diff
@@ -223,6 +223,15 @@ <h2 class="sponsors__title">Bronze Sponsors</h2>
                 </a>
             </div>
         </div>
+        <div class="sponsors sponsors--infrastructure">
+            <h2 class="sponsors__title">Infrastructure Sponsors</h2>
+            <div class="sponsors__content">
+                <a class="sponsors__link" href="https://vercel.com?utm_source=[bevy]&utm_campaign=oss">
```

Comment:
This just needs to say `https://vercel.com?utm_source=bevy&utm_campaign=oss` (no square brackets required for the utm parameter)

Reply:
Ah, thanks! There's a couple places where we should correct this.